R/submit-model.R

```diff
@@ -323,42 +303,35 @@ submit_models.character <- function(
   all_ext <- tools::file_ext(.mods) %>% unique()
 
   if (length(all_ext) != 1) {
-    stop(paste(
-          "If passing character vector to submit_models(), must pass all the same type of file.",
-          glue("Got files with {length(all_ext)} different extensions: {paste(all_ext, collapse=', ')}"),
-          sep = " "
-         ))
+    # .yaml and .yml OR .mod and .ctl can coexist, but otherwise must all be the same.
```

Comment:
per screenshare review - this extra carefulness is likely not needed as the map/read_model block will protect the user against invalid inputs

Reply:
it would be good to add a test around demonstrating that we can pass these multiple types successfully